apps/src/applab/dropletConfig.js

```diff
@@ -1099,6 +1100,17 @@ export var blocks = [
   }
 ];
 
+if (experiments.isEnabled(experiments.APPLAB_DATASETS)) {
+  blocks.push({
+    func: 'getColumn',
+    parent: api,
+    category: 'Data',
+    paletteParams: ['table', 'column'],
+    params: ['"mytable"', '"mycolumn"'],
+    nativeIsAsync: true
```

Comment:
Should this also be `type: 'value'` since it's a getter?

Reply:
Good catch, thanks!